lib/matplotlib/_mathtext_data.py

```diff
@@ -1016,7 +1018,7 @@
     'leftparen'                : 40,
     'rightparen'               : 41,
     'bigoplus'                 : 10753,
-    'leftbrace'                : 124,
+    'leftbrace'                : 123,
```

Comment:
This used to be a vertical bar.

Reply:
Not sure I understand the comment. Yes, it was wrong and is now correct?

Or are you saying that people have used `\leftbrace` to get a vertical bar? (Which I think was wrong until recently...)